guides/image-sources/provisioning-installer-bootdisk-fullhost.plantuml

```diff
@@ -0,0 +1,63 @@
+@startuml
+
+!include foreman.pstyle
+!$networkboot = 0
+
+title Installer-based provisioning with full-host boot disk on bare metal
+
+actor       User
+participant "Provisioned\nHost" as Host
+participant "Foreman" as Foreman
+participant "Foreman\nProxy" as Proxy
+participant DNS
+!if ($puppet)
+participant "Puppet\nserver" as Puppet
+!endif
+
+note over Host : Powered off
+
+== Create host in Foreman ==
+
+User -> Foreman : Create host
+User -> Foreman : Assign a static IP address
+group Templates [*PXE* installer boot]
+    Foreman -> Proxy : Render boot-loader configuration files
+end
+Proxy -> Proxy : Deploy bootloader configuration files\nto ""/var/lib/tftpboot""
+Foreman -> Proxy : Command to download installer kernel\nand initial RAM disk
+Proxy -> Proxy : Download kernel and initial RAM disk
+Foreman -> Proxy : Create DNS records
+Proxy -> DNS : Forward DNS records
+Proxy -> Foreman : Report status of TFTP, DHCP & DNS
+Foreman -> User : Host is created\n(build mode enabled)
+
+User -> Foreman : Download the boot-disk ISO of the host
+User -> User : Write the boot-disk ISO\nto an external storage device
```

Comment:
Do we need to mention it? I would be fine just to have `Host -> Host : Boot from the Full host iso` on the line 42.

Reply:
Yes, we do because the step is relevant from user's perspective.